resolwe/flow/managers/workload_connectors/base.py

```diff
@@ -0,0 +1,31 @@
+""".. Ignore pydocstyle D400.
+
+===================================
+Abstract Workload Manager Connector
+===================================
+
+"""
+
+
+class BaseConnector(object):
+    """The abstract base class for workload manager connectors.
+
+    The main :class:`~resolwe.flow.managers.dispatcher.Manager` instance
+    in :data:`~resolwe.flow.managers.manager` uses connectors to handle
+    communication with concrete backend workload management systems,
+    such as Celery and SLURM. The connectors need not worry about how
+    jobs are discovered or how they're prepared for execution; this is
+    all done by the manager.
+    """
+
+    def submit(self, data, dest_dir, argv, verbosity=1):
```

Comment:
Also add a method for cancelling jobs, as this is the next feature we have to support.

Reply:
I think it would be better to add that when we actually get around to implementing it. Having just a stub method which nobody can use seems a bit pointless, especially because we don't have any binary compatibility to worry about.